### PR TITLE
Update ug with add tag command

### LINKED ISSUE #125: Update UG to include `addtag` command

### PR DESCRIPTION
**Pull request by Arixeyeion**
Closes #125

### CODE REVIEW

**Review by KohGuanZeh — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -318,5 +337,6 @@ Action | Format, Examples
 **Schedule** | `schedule INDEX s/START_DATETIME e/END_DATETIME`<br> e.g., `schedule 1 s/23-02-2024T16:00 e/23-02-2024T17:00`
 **Unschedule** | `unschedule INDEX`<br> e.g., `unschedule 1`
 **Remark** | `remark INDEX r/REMARK`<br> e.g., `remark 1 r/Plans to own a house by age 35`
+**Add Tag** | `addtag INDEX t/TAG`<br> e.g., `addtag 1 t/PRUSaver` <br> `addtag 1 t/PRUSaver t/PRUGrowth`
```
Comment (Javiery3889) [inline docs/UserGuide.md:340]: This should match with the command format in the `addtag` section above:
```suggestion
**Add Tag** | `addtag INDEX t/TAG…`<br> e.g., `addtag 1 t/PRUSaver` <br> `addtag 1 t/PRUSaver t/PRUGrowth`
```

Comment (Arixeyeion) [reply docs/UserGuide.md:340]: ok I will change it

**Review by KohGuanZeh — APPROVED** [review]
LGTM

**Review by Javiery3889 — APPROVED** [review]

**Pull request merged**